src/prefect/utilities/compatibility.py

```diff
@@ -21,3 +20,17 @@ def nullcontext() -> Iterator[None]:
 else:
 
     from contextlib import nullcontext  # noqa: F401
+
+
+# Provide 3.6/3.7 `call` with `.args` and `.kwargs`
+
+
+def Call(call):
+    # Takes a `unittest.mock.call` and adds args/kwargs properties
+
+    if sys.version_info < (3, 8):
+        # Properties were added in 3.8
+        call.args = call[1]
+        call.kwargs = call[2]
+
+    return call
```

Comment:
I am not enthused about this but it 
- feels clunky no matter how we handle this
- it's only used in tests
- I think it improves test readability at the expense of a more confusing util

🤷